R/teal_data-extract.R

```diff
@@ -0,0 +1,38 @@
+#'
+#' @section Subsetting:
+#' `x[names]` subsets objects in `teal_data` environment and limit the code to the necessary needed to build limited
+#' objects.
+#'
+#' @param names (`character`) names of objects included in `teal_subset` to subset
+#' @param x (`teal_data`)
```

Comment:
@llrs-roche `teal_data` is described but mostly the argument is called `teal_data` or `data`. In this case it is called `x` because we extend `[.` that uses `x` as the first argument. Funny enough, this `x` is not displayed in the documentation. It is only here to satisfy roxygen2 warnings : )

Reply:
Ok, `x` is in there. In our case I think `x (teal_data)` should be enought. Especially since it's a page about `teal_data` constructor and a way on how `[` S3 works. This is the style we have all over the place. Let me resolve this conversation